ArduPlane/servos.cpp

```diff
@@ -915,6 +915,11 @@ void Plane::set_servos(void)
     if (g2.ice_control.throttle_override(override_pct)) {
         // the ICE controller wants to override the throttle for starting, idle, or redline
         SRV_Channels::set_output_scaled(SRV_Channel::k_throttle, override_pct);
+#if HAL_QUADPLANE_ENABLED
+        if (quadplane.enabled()) {
+            quadplane.vel_forward.integrator = 0;
+        }
```

Comment:
```suggestion
        quadplane.vel_forward.integrator = 0;
```
Any need to check for enabled? 0 is the default it should not do any harm always setting.